CHANGES.md

```diff
@@ -43,6 +43,11 @@ OpenSSL 3.5
 
   *Tomáš Mráz*
 
+* The FIPS provider now performs a PCT on key import for RSA, EC and ECX.
```

Comment:
Does the order in this file need to match the order they were done in?

Reply:
I don't see any reason why.